CHANGELOG.asciidoc

```diff
@@ -36,6 +36,10 @@ Notes:
 [float]
 ===== Features
 
+* Adds support for Amazon SQS queues via `aws-sdk` instrumentation that
+  partially implements the https://github.com/elastic/apm/blob/master/specs/agents/tracing-instrumentation-messaging.md[APM messaging spec],
+  and adds `queue.latency.min.ms`, `queue.latency.max.ms`, and `queue.latency.avg.ms` 
+  metrics for SQS queues. 
```

Comment:
Q: How does a user view these? I'm showing my ignorance with metrics here.
Perhaps you could add a comment to the PR showing a Kibana screenshot looking at those metrics?

Reply:
Good question -- it looks like metrics come to rest in  the `apm-[version]-metric-...` elasticsearch indices, which should be query-able via the `apm*` or `apm-*` kibana index pattern. Screenshot added the the PR description above.